packages/expand-button/expand-button.scss

```diff
@@ -81,30 +79,31 @@
 
     &--icon-only {
         .jkl-expand-button__arrow {
-            margin: jkl.$spacing-2xs jkl.$spacing-xs;
+            margin: jkl.$spacing-4 jkl.$spacing-8;
         }
     }
 
     &__arrow {
         @include jkl.motion;
 
         display: inline-block;
-        margin-left: jkl.$spacing-xs;
-        margin-bottom: jkl.rem(-3px);
+        margin-left: jkl.$spacing-2;
+        margin-bottom: jkl.rem(-8px);
 
-        .jkl-expand-button--expanded & {
-            margin-bottom: 0;
+        .jkl-animated-vertical-arrows__slider[data-show="up"] {
+            // For å få pila på linje med teksten
+            transform: translate3d(0, -56%, 0);
```

Comment:
Her er et første eksempel på at vi må jobbe litt rundt den innebyggede lufta for å få selve ikonet på linje med teksten. Var det sånn at vi så for oss en variant av disse uten innebygget luft?

Ser vi får det samme problemet på Button, der pila brått har blitt en del mindre visuelt.

Reply:
Jeg hadde vel sett for meg en variant der ikonet var plassert slik i boksen sin at det alignet riktig med tekst. En versjon uten luft vil sikkert funke for mange av dem, men noen av dem vil nok måtte justeres litt opp eller ned. Jeg måtte justere posisjonen litt i AlertMessage også pga samme issue.